cmd/multiFaReduce/multiFaReduce.go

```diff
@@ -6,14 +6,27 @@ package main
 import (
 	"flag"
 	"fmt"
-	"log"
-
+	"github.com/vertgenlab/gonomics/bed"
 	"github.com/vertgenlab/gonomics/fasta"
+	"log"
 )
 
-func mfaReduce(inFilename, outFilename string) {
+func mfaReduce(inFilename, outFilename, bedFilename, chrom string) {
 	aln := fasta.Read(inFilename)
-	answer := fasta.SegregatingSites(aln)
+	var answer []fasta.Fasta
+	var answerBedPos [][]int // answerBedPos is [][]int not [][]bed.Bed to avoid circular dependencies, aka trying to import bed package in fasta package
```

Comment:
Would it perhaps make more sense to put this in a different package to avoid the circular dependencies? Depending on its usage, it could be justified in popgen or bed, or perhaps something else